julia/sync.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Add change the shabang.
```suggestion
#!/bin/sh
```